ftw/news/upgrades/20200527193316_remove_plone_app_referenceablebehavior/types/ftw.news.News.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0"?>
+<object name="ftw.news.News"
+        meta_type="Dexterity FTI"
+        xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+        i18n:domain="ftw.news" >
+
+    <property name="title" i18n:translate="">News</property>
+    <property name="description" i18n:translate=""></property>
+    <property name="icon_expr"></property>
+    <property name="allow_discussion">False</property>
+    <property name="global_allow">False</property>
+    <property name="filter_content_types">True</property>
+    <property name="allowed_content_types">
+        <element value="ftw.simplelayout.TextBlock" />
+        <element value="ftw.simplelayout.FileListingBlock" />
+        <element value="ftw.simplelayout.MapBlock" />
+        <element value="ftw.simplelayout.VideoBlock" />
+        <element value="ftw.simplelayout.GalleryBlock" />
+    </property>
+
+    <property name="schema">ftw.news.contents.news.INewsSchema</property>
+    <property name="klass">ftw.news.contents.news.News</property>
+    <property name="add_permission">ftw.news.AddNews</property>
+
+    <property name="behaviors">
+        <element value="ftw.simplelayout.interfaces.ISimplelayout" />
+        <element value="plone.app.dexterity.behaviors.metadata.IBasic"/>
+        <element value="collective.dexteritytextindexer.behavior.IDexterityTextIndexer" />
+        <element value="plone.app.content.interfaces.INameFromTitle" />
+        <element value="plone.app.dexterity.behaviors.metadata.ICategorization" remove="True" />
+        <element value="ftw.keywordwidget.behavior.IKeywordCategorization" />
+        <element value="plone.app.dexterity.behaviors.metadata.IPublication" />
+    </property>
+
+    <property name="default_view">@@simplelayout-view</property>
+    <property name="default_view_fallback">False</property>
+    <property name="view_methods">
+        <element value="@@simplelayout-view"/>
+    </property>
+
+    <alias from="(Default)" to="(dynamic view)"/>
+    <alias from="edit" to="@@edit"/>
+    <alias from="sharing" to="@@sharing"/>
+    <alias from="view" to="(selected layout)"/>
+
+    <action
+        action_id="view"
+        title="View"
+        category="object"
+        condition_expr=""
+        url_expr="string:${object_url}"
+        visible="True">
+        <permission value="View"/>
+    </action>
+
+    <action
+        action_id="edit"
+        title="Edit"
+        category="object"
+        condition_expr=""
+        url_expr="string:${object_url}/edit"
+        visible="True">
+        <permission value="Modify portal content"/>
+    </action>
+
+</object>
```

Comment:
You only have to add the delta otherwise it will overwrite policy level customization. 
```suggestion
<?xml version="1.0"?>
<object name="ftw.news.News">
  <property name="behaviors" purge="False">
    <element value="plone.app.referenceablebehavior.referenceable.IReferenceable" remove="True"/>
  </property>
</object>
```